### ISSUE TITLE
The search is not working 

### ISSUE THREAD

**Issue by flynnridercode** [BeamlakAschalew/flixquest#69]
### Steps to reproduce

Search something 

### Expected behavior

The movie/series should appear

### Actual behavior

It shows error 404 not found

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by BeamlakAschalew** [comment]
Have you enabled the "Use proxy for TMDB fetch" option in the app settings? If so the app crashes there for some reason, didn't really got the time to check what causes it but turn off that

**Comment by flynnridercode** [comment]
I have it turned on but when I turn it off no movies even on the homepage load

**Comment by BeamlakAschalew** [comment]
> I have it turned on but when I turn it off no movies even on the homepage load

Yes, tmdb is blocked in some countries that's why, use VPN instead
